src/test/java/jenkins/security/plugins/ldap/CaptureLogRule.java

```diff
@@ -0,0 +1,167 @@
+package jenkins.security.plugins.ldap;
+
+
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.logging.Logger;
+
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertThat;
+
+public class CaptureLogRule extends Handler implements TestRule {
```

Comment:
Would prefer that any useful functionality be added to the standard `LoggerRule` and used from there.

Reply:
Ok, I'll submit a PR to augment LoggerRule.